curriculum.md

```diff
@@ -39,7 +39,41 @@ Here are some basic programming videos that are pretty good, but are done in Pyt
 
 ---
 
-## Week 3: React applications!
+## Week 3: More JavaScript and the DOM
+### Learning objective
+- Complementary JS skills for working with the DOM: 
+   - functions
+   - objects and arrays
+   - events
+   - math functions
+   - this/context
+   - jquery
+   - local storage
+
+### Resources
+- [Front-end JS (Beetroot)](https://beetroot.academy/en/courses/front-end/#module2)
+
+~~Homework~~ (20-40hr practice from Beetroot)
```

Comment:
I need to ask the Beetroot team if they have these resources in English or if they are able to translate them until autumn. I think it might be good to prepare some backup-resources in case it doesn't work out.

Reply:
Good point! Don't know why I added a strikethrough there ^^;